package.json

```diff
@@ -79,7 +82,7 @@
     "@types/jquery": "^3.3.31",
     "@types/node": "^10.14.19",
     "codelyzer": "^5.0.1",
-    "cypress": "4.0.1",
+    "cypress": "^4.0.1",
```

Comment:
Why did you change this? It didn't end up changing the package-lock.json (did you verify it didn't?), so it probably doesn't matter, but generally best to avoid unnecessary changes if possible.

Reply:
I didn't change it, I thought it was changed when I rebuilt it or something.  Can change it back if it's not important

Reply:
I feel like you had to have changed it :) but it's fine